### ISSUE TITLE
Rendering seems to be too slow on Chromium 

### ISSUE THREAD

**Issue by podborski** [fraunhoferhhi/omaf.js#5]
One can build Chromium with HEVC support enabled such as [here](https://chromium.woolyss.com/). However on win10 I had some issues with the performance of the renderer. 

Repackaging and playback of 4k HEVC works well (tested with the `me.html` test page and content created with [Nokia Creator](https://github.com/nokiatech/omaf/tree/master/Creator)).

However rendering seems a bit slow. After the MPD is loaded and renderer is initialised, it is hard to navigate through the scene.

**Comment by JWSon** [comment]
This issue depends on HEVC support of hardware acceleration on chromium of win10.

I tested rendering of webGL using [example ](https://threejs.org/examples/?q=equire#webgl_video_panorama_equirectangular)of three.js to see whether or not it's the only problem of our project.

***Experiment environments**
 - 4K test files(.mp4): avc and hevc
 - os: win10 
 - browser: chromium ver 74.0.3729.108. 
 - support codec of gpu
![gpu_support](https://user-images.githubusercontent.com/20334410/57089543-f3daa480-6d04-11e9-8a40-29088b31e30d.PNG)


***Execution time of teximage2D**
 - avc
![avc_animate](https://user-images.githubusercontent.com/20334410/57088105-dc4dec80-6d01-11e9-9224-2670bdc2f9c7.PNG)

- hevc
![hevc_animate](https://user-images.githubusercontent.com/20334410/57088253-28992c80-6d02-11e9-9fef-68ca358ad8de.PNG)

***Support of video decoding on GPU**

- avc
![avc_gpu](https://user-images.githubusercontent.com/20334410/57089420-b2e29000-6d04-11e9-9eef-f042a91d707f.PNG)

- hevc
![hevc_gpu](https://user-images.githubusercontent.com/20334410/57089436-bb3acb00-6d04-11e9-8457-a3495e69a396.PNG)

*Decoder

 - avc 
![avc_plaerproperties](https://user-images.githubusercontent.com/20334410/57090171-757f0200-6d06-11e9-90d7-75709d0734a4.PNG)

- hevc
![hevc_plaerproperties](https://user-images.githubusercontent.com/20334410/57090185-7e6fd380-6d06-11e9-93d2-37de90bbb2b3.PNG)



-> GPU supports hevc decoding but Chromium is not available. Referring to [this link](https://bbs.archlinux.org/viewtopic.php?id=244031), MojoVideoDecoder is a hardware decoder and FFmpegVideoDecoder is not clear. when Chromium lowers the hardware support resolution of avc to 1920x1080, the decoder of the 4k avc file changes to FFmpegVideoDecoder and the same problem occurred as hevc.
texImage2d function is delayed because it reads pixels from the CPU when hardware decoding is not possible ([reference link](https://hacks.mozilla.org/2013/04/the-concepts-of-webgl/)).

**Comment by podborski** [comment]
@JWSon I don't think it is related to video decoding at all, since the rendering is already too laggy when the video is paused and no decoder is involved. Please see the discussion in #8 for more details.

**Comment by podborski** [comment]
Since the issue seems to be related with Chromium webGL performance (under the hood) I think we can not fix it here.
